docs/blogs/reshare.html

```diff
@@ -0,0 +1,125 @@
+        <img src ="/imgs/reshare.png" alt="Reshare Flow" />
+        <p>The example combines <code>consensus::threshold_simplex</code>, <code>cryptography::bls12381::dkg</code>, and <code>p2p::authenticated</code> into a single demonstration that you can run end to end. Because every interaction in the reshare is in the log, each participant has a transcript of who proposed what, which shares were acknowledged, and how the next committee was formed.</p>
+
+        <h2>A refresher on <code>consensus::threshold_simplex</code> and <code>examples/vrf</code></h2>
+        <p>The <code>consensus::threshold_simplex</code> module utilizes BLS12-381 threshold signatures so that a group of validators can sign consensus messages. Each active validator has a share of the signing key. A threshold of these shares is required to sign a message.</p>
+        <p><a href="https://github.com/commonwarexyz/monorepo/tree/main/examples/vrf"><code>examples/vrf</code></a> showcases a continuous key reshare process to generate randomness, but with one flaw: participants trust a central "<a href="https://docs.rs/commonware-cryptography/latest/commonware_cryptography/bls12381/dkg/arbiter/struct.Arbiter.html">Arbiter</a>" to track commitments, acknowledgements, and reveals.</p>
+
+        <h2>Epoching</h2>
+        <p>Recently, <a href="https://x.com/B_Chou">Brendan Chou</a> added support for height-based epoching in <code>consensus::threshold_simplex</code>. This feature adds an <code>epoch</code> field to each consensus message, enabling an orchestrator to transition to a new epoch when a direct finalization for the last block in an epoch is observed. Epochs give us the perfect platform to perform progressive BLS key reshare to reconfigure the committee.</p>
+        <p>This approach was derived from <a href="https://arxiv.org/pdf/2506.14124"><i>From Permissioned to Proof-of-Stake Consensus</i></a> by Jovan Komatovic, Andrew Lewis-Pye, Joachim Neu, Tim Roughgarden and Ertem Nusret Tas.</p>
+
+        <h2>Piggybacking on Consensus</h2>
+        <p>To safely reconfigure the validator committee in a <code>consensus::threshold_simplex</code> engine, we use <code>cryptography::bls12381::dkg</code> to securely generate new shares for the next committee.</p>
+        <p>Each epoch begins with a threshold of validators acting as dealers and players for the next BLS shares. Dealers encrypt fresh shares to the upcoming committee (players), players acknowledge these shares, and anyone who misbehaves is flagged by participants. Those transcripts are bundled into a compact result and proposed alongside normal blocks.</p>
```

Comment:
nit: in two places you use `BLS` instead of `BLS12-381`. Consider using it consistently everywhere or switch to `BLS` after the first usage of the fully-qualified name